fireant/utils.py

```diff
@@ -23,6 +51,12 @@ def slice_first(item):
     return item
 
 
+a = 'a' \
```

Comment:
Isn't this always 'a'? What's the point of those ternary ifs?

Reply:
Looks like dead code

Reply:
not sure what this is. going to remove it